### ISSUE TITLE
Meta Manager UI Improvements

### ISSUE THREAD

**Issue by Kodylow** [fedimint/ui#506]
@sbddesign could use some help here with improving the meta manager design, especially on mobile. Currently has issues with overflowing / not multilining, but just generally I hate the key value thing we have it, I just want to show the json blob but potentially there's a better way of showing it:

![image](https://github.com/user-attachments/assets/fd1dedd6-04b8-47d2-af60-33459518df12)

**Comment by sbddesign** [comment]
I'll take it

**Comment by sbddesign** [comment]
I think there's a bunch of issues to break down here:
- Better layout desired on mobile
- Better way to present the default Fedimint meta fields
- Best way to show custom metakeys/values?

Here's a first crack. Overall, my gut reaction is that 99.9% of Fedimint Guardian users dont need the custom keys/values. They probably want to give their Federation a name and a cool avatar. So just expose those fields to them. The titles are self explanatory if you take out the underscores and put it in plain English (or "plain" whatever language it is translated into).

Also, a lot of this info probably belongs in the dev docs. Only people who actually bother to read the dev docs are going to benefit from writing in their own custom meta values. So just expose the defaults with a nice string field, date selector, and image preview, and link to docs (or some kind of expanded info thing) for people who want to dig deeper.

What's your initial reaction to something like this prototype?

[Prototype Demo](https://www.figma.com/proto/TcAV8bmhcaQAxiZE8hjWY9/Fedimint-UI?page-id=0%3A1&node-id=2-29&viewport=914%2C-601%2C0.45&t=HQXzPGA8kXgbd37B-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=2%3A29)

**Comment by sbddesign** [comment]
And we further discussed this and decided to try and put the proposals in their own tab. [See updated Figma frames](https://www.figma.com/design/TcAV8bmhcaQAxiZE8hjWY9/Fedimint-UI?node-id=13-338&t=IAKgtg8i1aZCSPda-1).

**Comment by Kodylow** [comment]
mertged
